docs/manual/users_and_groups.md

```diff
@@ -38,9 +41,10 @@ to weaken these into recommends-dependencies by setting
 
 ## Limitations
 
-At this time, rpm only supports the `u` and `g` directives of sysusers.d
-format and ignores others. If other directives are needed, the package
-will need to call systemd-sysusers with the correct arguments manually.
+At this time, rpm only supports the `u`, `g` and (since RPM 4.20) `m`
+directives of sysusers.d format and ignores others. If other
+directives are needed, the package will need to call systemd-sysusers
+with the correct arguments manually.
 
 ## Technical details
```

Comment:
The 'm' dependencies are different enough from user/group that they deserve an explanation and an example in the technical details section.